sql_to_code/parsers/alter_table/parser.py

```diff
@@ -1,32 +1,33 @@
-import re
+from pyparsing import *
 
 from .models import ForeignKey, Reference
 
-source_table_name_regex = re.compile('ALTER TABLE "(?P<table_name>\w+)"')
-foreign_key_name_regex = re.compile('ADD FOREIGN KEY \("(?P<foreign_key>\w+)"\)')
-result_table_name_regex = re.compile('REFERENCES "(?P<result_table_name>\w+)"')
-result_table_field_name_regex = re.compile(
-    'REFERENCES "\w+" \("(?P<result_table_field_name>\w+)"\);'
+table_name_schema = CaselessKeyword("alter table") + QuotedString('"')("table_name")
+foreign_key_schema = CaselessKeyword("add foreign key") + QuotedString(
```

Comment:
I kinda don't mind have whole phrase there, but what about split it to different tokens instead?
IMO:
* **READABILITY**: CaselessKeyword - from the class name I get it in the way that it should take only one word (not phrase)
* **USABILITY**: ADD, FOREIGN and KEY could be used in several different places (alter table, create table) in different context (with/without ADD keyword). In case we have one token for the whole phrase, we are not able to use single token and it forces us to re-create it in other places (not now, but definitely later it will).

Reply:
no need to do it now, it's overcomplicating the code. when we will decide to refactor this parsing schemas, then we can split it